migrations/versions/201610111427_44ef3c219621_allow_event_wide_track_conveners_.py

```diff
@@ -0,0 +1,47 @@
+"""Allow event-wide track conveners/reviewers
```

Comment:
I think it'd be better to merge the changes in the revision where those tables are initially created.